smoke/features/persistent-jenkins.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Deploy Jenkins with persistent volume on openshift using template based install
```

Comment:
So best as I can tell, this test assumes the persistent volumes are created prior to the invocation of the test.

If that is correct @jitendar-singh I believe that should be explicitly called out here and in the other persistent*.feature files.

Reply:
Or, ideally, you replicate the NFS based PV setup we do in the current extended tests at https://github.com/openshift/origin/blob/master/test/extended/builds/pipeline_jenkins_e2e.go#L60-L72

Then we could be on a path of removing those persistent template tests in openshift/origin and replace them with CI invocations of your new smoke tests.

Reply:
Thanks @gabemontero I will work this next.